src/components/title/create/title-create.js

```diff
@@ -61,14 +64,58 @@ export default class TitleCreate extends Component {
       >
         <FormattedMessage id="ui-eholdings.actionMenu.cancelEditing" />
       </Button>
-    ) : null;
-  }
+    );
+  };
+
+  renderFirstMenu = () => {
+    const { onCancel } = this.props;
+
+    return (
+      <FormattedMessage
+        id="ui-eholdings.label.icon.closeX"
+        values={{ paneTitle }}
```

Comment:
I believe `paneTitle` should be destructured

Reply:
it is not a prop
It is declared right after the imports